EndlessClient/UIControls/ScrollBar.cs

```diff
@@ -160,12 +161,17 @@ private void OnScrollButtonDragged(object btn, EventArgs e)
             else if (y > scrollArea.Height - _scrollButton.DrawArea.Height)
                 y = scrollArea.Height - _scrollButton.DrawArea.Height;
 
-            _scrollButton.DrawPosition = new Vector2(0, y);
+            var dif = (CurrentMouseState.ScrollWheelValue - PreviousMouseState.ScrollWheelValue) / -160;
```

Comment:
If 160 is the height of the scrollbar, different scrollbars have different heights (chat vs shop dialog vs account create dialog), so it can't be hard-coded

Reply:
I also don't see where this variable is being used